### ISSUE TITLE
Desktops apps dont work properly etc

### ISSUE THREAD

**Issue by spl1inter** [Alexpux/MINGW-packages#4122]
Nice job. However few things:

1) Many of these packages don't work properly for native desktop?
For example your x64 VLC is missing default interface buttons. But skins work ok.
Inkscape fails to launch, internal error, closes.
QT5 is completely broken. Wrong prefix (non-portable):
```
I:\Download\MinGW-64-souorce\QupZilla-2.2.6>qmake --version
QMake version 3.1
Using Qt version 5.11.1 in C:/repo/mingw-w64-qt5/pkg/mingw-w64-x86_64-qt5/mingw6
4/lib
```
QT build prefix (as all other MinGW64 apps) should always have "HD-neutral" prefix like "/MSYS/mingw64" or similar.

Notice you build python and build readline, but don't go through basic trouble to make python-readline lol, also lol need to drag whole MSYS2 runtime to run patch-diff. RIP GnuWIN? 

Stuff like this.

PS I already take your work and do MinGW64 native groff and less. Looking at 'Patch' but it is poorly made with many POSIX calls not actually needed to just modify one file against the contents of another <facepalm>.

**Comment by Alexpux** [comment]
@spl1inter about issues:
1. About VLC I dont know what button you mean, maybe it not enables for mingw somehow
2. Qt5 is not broken. There are two possibilities:
  - not installed correcly with pacman - in most cases reinstalling helps
  - if you unpack manually then you need run qtbinpatcher yourself to fix paths.
3. Readline module for MINGW Python is desabled because it broken for windows

**Comment by spl1inter** [comment]
Hi Alexpux,

1. The VLC main (default no-skin) interface buttons for play pause etc.
2. I remember now, QT has this stupid need to hardcode full DOS path, really dumb. I worked around in QT4 by editing config by hand to correct set prefix to driveless "/mingw". Don't feel like building from source again and "qtbinpatcher --qt-dir=/MSYS2/mingw64" seems to fix both QT issues I raise. Some concern that QT again hardcode the full DOS version of this path which will break again if I ever try to run it from a DVD but this is minor issue..
3. Yes I had to do some digging also, to find a working readline when I built my pervious 32-bit system several years back. Since you included it I was assuming working, sorry my mistake.
4. If you know its broken should remove from repo.

**Comment by Alexpux** [comment]
@spl1inter if package is broken it is not about removing it. Need to fix and upload new version

**Comment by spl1inter** [comment]
Temporary remove only. Perhaps a note in place stating temporarily unavail. This way users save time as they know if they want working version need to roll their own.

**Comment by Alexpux** [comment]
I have thousands of packages in repo and part of them are broken so I will not remove just because in one time it broken because I can’t verify every package and my time limited. So if you disagree - I don’t care

**Comment by spl1inter** [comment]
cool, what I wanted to know.

**Comment by Alexpux** [comment]
@spl1inter about VLC - what interface you use? Qt?
I have:
![default](https://user-images.githubusercontent.com/1898153/43404264-b88134e4-941f-11e8-9421-24071228fc0a.png)

**Comment by spl1inter** [comment]
Yes QT5 all defaults. Reinstall after running qtbinpatcher seems to fix for me as well.
I will make a note, since QT4.7 didn't seem to include this utility.
